File: platforms/android/farwolf.weex/src/main/java/com/farwolf/weex/module/WXFLogModule.java
package com.farwolf.weex.module;

import android.util.Log;

import com.farwolf.weex.base.WXModuleBase;
import com.farwolf.weex.util.HotRefreshManager;
import com.taobao.weex.annotation.JSMethod;

import java.util.HashMap;

public class WXFLogModule extends WXModuleBase {


    @JSMethod
    public void log(HashMap param)
    {
        String msg=param.get("msg")+"";
        String level="info";
        if(param.containsKey("level")){
            level=param.get("level")+"";
        }
        Log.i("weexplus",msg);
//        HotRefreshManager.getInstance().send("log:"+msg);
        HotRefreshManager.getInstance().send("log:"+level+"level:"+msg);

    }
}

File: platforms/ios/weexplus/farwolf.weex/farwolf.weex/module/WXFLogModule.m
//
//  WXFLogModule.m
//  AFNetworking
//
//  Created by 郑江荣 on 2019/4/11.
//

#import "WXFLogModule.h"
#import "farwolf.h"
#import "Weex.h"

#import <WeexSDK/WXEventModuleProtocol.h>
#import <WeexSDK/WXModuleProtocol.h>

@implementation WXFLogModule
WX_EXPORT_METHOD(@selector(log:))
-(void)log:(NSMutableDictionary*)param{
    NSString *msg=param[@"msg"];
    
    NSString *level=level;
    if(param[@"level"]){
        level=param[@"level"];
    }
    NSLog([@"weexplus:" add:msg]);
    NSString *m=[[[@"log:" add:level]add:@"level:"]add:msg];
//    NSString *m=[@"log:" add:msg];
   
    [[Weex getRefreshManager] send:m];
}
@end
